lib/public/components/common/selection/infoLoggerButtonGroup/infoLoggerButtonGroup.js

```diff
@@ -0,0 +1,52 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+
+import { frontLink } from '../../navigation/frontLink.js';
+import { dropdown } from '../../popover/dropdown.js';
+import { iconCaretBottom } from '/js/src/icons.js';
+import { h } from '/js/src/index.js';
+import { getInfologgerLink } from '../../../../services/externalRouting/getInfologgerLink.js';
+import { CopyToClipboardComponent } from './CopyToClipboardComponent.js';
+
+/**
+ * Renders a button group that links to a specific details page based on the given parameters,
+ * and a dropdown menu with options to copy the link or access the related infologgers.
+ *
+ * @param {Object} parameters the parameters that specify what links to display for the details page and the infologgers.
+ * @return {Component} the infologger button group component
+ */
+export const infoLoggerButtonGroup = (parameters) => {
+    const { flp: flpInfologger, epn: epnInfologger } = getInfologgerLink(parameters);
+    const { environmentId, runNumber } = parameters;
+    const key = runNumber ? { runNumber } : { environmentId };
+    const value = runNumber || environmentId;
+    const page = runNumber ? 'run-detail' : 'env-details';
```

Comment:
I think that here you are coupling a bit too much your component with specific pages.
I would say that you should have a component for example `identifierColumndAndActionsDropdown` that takes as a arguments the page, the link content (value for you), the parameters (not key, parameters), and a list of dropdown entries. 
In this component, you display the component you have here, and you build the dropdown options with the copy button and the list of given dropdown entries.

Then you will have two calls, one for run and one for envs.

Because here you are really adding too much logic in this components (info about runs, info about environment), making it quite hard to maintain.